stl/inc/ranges

```diff
@@ -6681,14 +6681,8 @@ namespace ranges {
                             "cannot advance chunk_view iterator past end");
                     }
 #endif // _ITERATOR_DEBUG_LEVEL != 0
-
-                    // Per to-be-filed LWG issue (See GH-2995)
-                    if constexpr (sized_sentinel_for<_Base_sentinel, _Base_iterator>) {
-                        _Missing = _RANGES advance(_Current, _Count * _Off, _End);
-                    } else {
-                        _Current += static_cast<difference_type>(_Count * (_Off - 1));
-                        _Missing = _RANGES advance(_Current, _Count, _End);
-                    }
+                    _RANGES advance(_Current, _Count * (_Off - 1));

```

Comment:
I think this should be `_Current += _Count * (_Off - 1);`, no? What's the reasoning behind using `_RANGES advance`?

Reply:
> What's the reasoning behind using `_RANGES advance`?

That's what LWG-3880 said; but I agree with you `operator+=` is better here.